lib/data_filter/filter_set.rb

```diff
@@ -92,9 +92,9 @@ def create(&block)
       #   to filter
       # @option opts [Object] :by the value we want to use when
       #   filtering the data item
-      def like_filter(field_sym, opts = {})
+      def like_filter(field_sym, opts = {}, normalize_regex = /[^\w\s]/)
```

Comment:
I'm totally ok with it being another option. I'll update that, the doc block and the examples.